src/caret_analyze/architecture/architecture_loaded.py

```diff
@@ -1598,7 +1613,19 @@ def __init__(
         paths: list[NodePathStruct] = []
 
         if callbacks is not None:
+            skip_count = 0
+            import sys
+            construction_order = {'min': sys.maxsize, 'max': 0}
```

Comment:
I suggest changing `construction_order` (dict) to `max_ignored_construction_order` (int) because only `construction_order['max']` is used.

```suggestion
            max_ignored_construction_order = 0
```

Reply:
consent.